src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,25 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+import java.util.List;
+
+
+/**
+ * Lists all persons in the address book to the user.
```

Comment:
Class comment does not match description of the command!

Reply:
The description is the same as that of the ListCommand.java class

Reply:
> The description is the same as that of the ListCommand.java class

Fixed in latest commit.